bike_rent/models/bike_rent.py

```diff
@@ -11,7 +12,7 @@ class BikeRent(models.Model):
 
     bike_id = fields.Many2one(
         'product.product',
-        string='Bike Model Name',
+        string='Bike Rent Product',
```

Comment:
Recommendation would be to stick to just `Bike`, if you want to leave a more descriptive comment, use `help=''`